test/e2e-tests-auction.sh

```diff
@@ -0,0 +1,25 @@
+#!/bin/bash
+set -e
+
+cd dex-contracts/
+
+# Place 6 orders in current Auction (accountId, buyToken, sellToken, minBuy, maxSell)
+truffle exec scripts/sell_order.js 1 2 3 12 12
```

Comment:
Would it make sense to use a different accountId for some of these trades?

Reply:
It would make sense especially for checking updated balances